source/dub/internal/utils.d

```diff
@@ -198,7 +198,8 @@ void runCommands(in string[] commands, string[string] env = null)
 		Pid pid;
 		pid = spawnShell(cmd, stdin, childStdout, childStderr, env, config);
 		auto exitcode = pid.wait();
-		enforce(exitcode == 0, "Command failed with exit code "~to!string(exitcode));
+		enforce(exitcode == 0, "Command ``" ~ cmd
+			~ "'' failed with exit code "~to!string(exitcode));
 	}
```

Comment:
Why two backticks and then two single quotes?
Usually one is enough and typically you end the quotation with the same character, no?

Reply:
Right, (two backticks, then two single quotes) is uncommon style. It's found in old GNU manuals and in AsciiDoc. I've merely tried to be distinct from any quotes within the shell command itself.

Maybe the best is to avoid quoting altogether, and rewrite the error as such?
```
Command failed with exit code 4: xcopy /s /e /y C:\... C:\...
```

I'll re-push with that quoteless style.

I'd also be happy with any other quoting style, feel free to pick one, although dub output generally avoids quotes.